redirects/mstg.md

```diff
@@ -2,6 +2,11 @@
 
 title: MSTG Redirect
 permalink: /mstg
+redirect_from:
+  - /mstg/
+  - /mstg
```

Comment:
Since this is the permalink, no need for this one :)

Reply:
fixed, thsanks!